### ISSUE TITLE
Audit specific pages with high TTI

### ISSUE THREAD

**Issue by brandonrapp** [department-of-veterans-affairs/va.gov-team#3961]
Jennifer Strickland identified some pages with TTI times of 12s which impact accessibility tools.

The issues I'm seeing with connectivity that may have to do with performance is that VoiceOver in Safari and NVDA in Firefox are not able to connect to the Skip to Content link. For VO, if I wait 20-30 seconds, then I am able to tab to the link. When I ran performance audits and investigated by frame, it looks like the page content between header and footer is loading for a long time. When I investigated the page structure I noticed all the scripts in the <head> and wondered if there was a reason they were all there instead of beneath the <footer>.  If it is possible to evaluate what resources are needed immediately, perhaps some could be put beneath the footer to optimize perf.

Actions:
We need to profile this page and come up with ideas for improving TTI https://web.dev/interactive/?utm_source=lighthouse&utm_medium=unknown
we should note jennifer’s suggestion to defer all  / more of our js.

Pages to profile: 
https://staging.va.gov/disability/view-disability-rating/rating/
 https://staging.va.gov/disability-benefits/apply/form-0996-higher-level-review

Full Details in the slack thread here - https://dsva.slack.com/archives/CQH357ZTP/p1575306412187600
